app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,22 @@
  */
 .cancel {
   text-align: center;
+}
+
+body {
+ background-image: image_url("");
+ background-size: cover;
+ background-repeat: no-repeat;
+ background-position: center top;
+ background-position: 0px 45px;
+}
 
+.welcome {
+  background:rgba(255,255,255,0.5);
+  padding: 10px;
+}
+.footer {
+  background-color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.